libpod/options.go

```diff
@@ -1041,13 +1043,28 @@ func WithLogPath(path string) CtrCreateOption {
 		if path == "" {
 			return fmt.Errorf("log path must be set: %w", define.ErrInvalidArg)
 		}
-
-		ctr.config.LogPath = path
+		if strings.HasSuffix(path, "/") || isDirectory(path) {
```

Comment:
The suffix check means you are going to create the directory (and all parents) even when it does not already exists? I don't think this is a good default. I think it should be safest to assume the directory path must exists.